docs/apache-hive-catalog.md

```diff
@@ -116,6 +116,10 @@ The following table lists the data types mapped from the Hive catalog to Graviti
 | `struct`                    | `struct`            | 0.2.0         |
 | `uniontype`                 | `uniontype`         | 0.2.0         |
 
+:::info
+Since 0.5.0, the data types other than listed above are mapped to Gravitino `unparsed` type that represents an unresolvable data type from the Hive catalog.
```

Comment:
Please add a link to unparsed

Reply:
Links need to be added to other documents where the `unparsedType`  is mentioned.